accounts/pkg/storage/cs3.go

```diff
@@ -165,13 +201,47 @@ func (r CS3Repo) WriteGroup(ctx context.Context, g *proto.Group) (err error) {
 	return nil
 }
 
-// LoadGroup loads a group via cs3 by id and writes it to the provided group
+// LoadGroup loads A group via cs3 by id and writes it to the provided group
```

Comment:
did you replace `a` with `A`?

Reply:
Refactoring in goland with too little intention did, yes. :-( same for `g` to `G`. Fixed them. Thanks for spotting!